src/Sql/dbo/Stored Procedures/OrganizationUser_DeleteByIds.sql

```diff
@@ -25,22 +25,74 @@ BEGIN
         EXEC [dbo].[SsoUser_DeleteMany] @UserAndOrganizationIds
     END
 
-    DELETE CU
-    FROM
-        [dbo].[CollectionUser] CU
+    
+    DECLARE @BatchSize INT = 100
+
+-- Delete SSO Users
+WHILE @BatchSize > 0
```

Comment:
Sorry I missed this before, this `WHILE` block just needs to be indented to match the others